### ISSUE TITLE
Font selection support

### ISSUE THREAD

**Issue by xakod** [Profactor/cv-plot#32]
I see cv-plot uses cv::FONT_HERSHEY_SIMPLEX while it doesn't support non-ascii characters (so we cannot use non-ascii characters with cv-plot at the moment) meanwhile COMPLEX font supports utf-8 opencv/opencv#17980 and if we want more, opencv provides TrueType opencv/opencv#18760.

 Why not to provide some kind of selection for internal fonts?

**Comment by wpalfi** [comment]
Yes, that would be nice. I have no experience with opencv fonts. Feel free to propose a fix if it is simple and always works with any opencv installation!
